### PR TITLE
DOC: add examples to interpolate.lagrange docstring

### PR DESCRIPTION
**Pull request by ArkadiyD**

### CODE REVIEW

**Comment by person142** [comment]
Why close the old PR and open a new one?

**Comment by Arcady27** [comment]
Sorry, did it by mistake

File: scipy/interpolate/interpolate.py [hunk 1]
```diff
@@ -61,8 +61,25 @@ def lagrange(x, w):
...
+    >>> y = [1, 8, 27]
+    >>> poly = lagrange(x, y)
+    
+    Since there are only 3 points, Lagrange polynomial has degree 2.
+
+    >>> from numpy.polynomial.polynomial import Polynomial
+    >>> print (Polynomial(poly))
+    poly([  6. -11.   6.])
+            
+    Polynomial coefficients are in order of increasing degree, so Lagrange polynomial is 6 - 11x + 6x^2
```
Comment (pv) [inline scipy/interpolate/interpolate.py]: Maybe use some other polynomial where this is not ambiguous?
Note also that the line is somewhat long (recommend to keep <80 chars per line).

**Comment by ev-br** [comment]
Thanks @Arcady27 for your contribution! It has been merged as a part of gh-7642, along with some further tweaks. Keep them coming :-).

**Pull request closed without merging**